modules/op-using-ssh-authentication-in-git-type-tasks.adoc

```diff
@@ -1,11 +1,30 @@
 // This module is included in the following assemblies:
-// * secure/authenticating-pipelines-using-git-secret.adoc
+// * secure/authenticating-pipelines-repos-using-secrets.adoc
 
 [id="op-using-ssh-authentication-in-git-type-tasks_{context}"]
-= Using SSH authentication in git type tasks
+= Using Git SSH authentication in tasks
 
-When invoking Git commands, you can use SSH authentication directly in the steps of a task. SSH authentication ignores the `$HOME` variable and only uses the user's home directory specified in the `/etc/passwd` file. So each step in a task must symlink the `/tekton/home/.ssh` directory to the home directory of the associated user.
+You can invoke directly Git commands in the steps of a task and use SSH authentication, but an additional step is required.
 
-However, explicit symlinks are not necessary when you use a pipeline resource of the `git` type, or the `git-clone` task available in the Tekton catalog.
+{pipelines-shortname} provides the SSH files in the `/tekton/home/.ssh` directory and sets the `$HOME` variable to `/tekton/home`. However, Git SSH authentication ignores the `$HOME` variable and uses the home directory specified in the `/etc/passwd` file for the user. Therefore, a step that uses Git command must symlink the `/tekton/home/.ssh` directory to the home directory of the associated user.
+
+For example, if the task runs as the `root` user, the step must include the following command before Git commands:
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: example-git-task
+spec:
+  steps:
+    - name: example-git-step
+#     ...
+      script:
+        ln -s $HOME/.ssh /root/.ssh
+#     ...
+----
+
+However, explicit symlinks are not necessary when you use a pipeline resource of the `git` type or the `git-clone` task available in the Tekton catalog.
 
 As an example of using SSH authentication in `git` type tasks, refer to link:https://github.com/tektoncd/pipeline/blob/main/examples/v1/taskruns/authenticating-git-commands.yaml[authenticating-git-commands.yaml].
```

Comment:
This is even further out of scope for this review, so I'm only pointing it out this time, but this is another instance of a problematic link to a GitHub-hosted repository.

Reply:
Again, I believe the repo is maintained by Red Hat engineers. There are many links to this and other similar repos in this doc set and removing them would be a major problem. As far as I am aware, the intend of the guidance is to disallow repos that we don't control and that therefore can be volatile.

Reply:
The examples could still change out from under us and not necessarily have been QE verified, particularly for the docs version they might be appearing in (especially if just targeting the `main` branch like this example is).

Reply:
I have requested Pipelines QE feedback on these examples.